components/form/fields/TextField.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+    TextField as MuiTextField,
+    FormControl,
+    FormHelperText
```

Comment:
You imported this in, but never used it. Let's add the error support into the fields